images/image-builder-script/builder.sh

```diff
@@ -42,7 +42,7 @@ fi
 
 echo "Executing builder..."
 PUSHED_IMAGE=$(cd "$SCRIPT_DIR" && \
-    go run . --build-dir "${PROJECT_DIR}"/"${BUILD_DIR}" "$@")
+    go run . --build-dir "${PROJECT_DIR}"/"${BUILD_DIR}" $@)
```

Comment:
hmm i think you want to keep the double quotes here, otherwise the arguments with spaces will get treated as different arguments

example:

```
images/image-builder-script/builder.sh "foo bar"
```

will be interpreted as

```
go run . --build-dir "/home/mvalais/code/cert-manager/_build" foo bar
```

instead of

```bash
go run . --build-dir "/home/mvalais/code/cert-manager/_build" "foo bar"
```

```suggestion
    go run . --build-dir "${PROJECT_DIR}"/"${BUILD_DIR}" "$@")
```

Reply:
but what happens when the provided value is eg. "--test1 value1 --test2 value2"

Reply:
Thanks, this seems to work!